job-utilities/job-router/lambda_function.py

```diff
@@ -0,0 +1,75 @@
+"""
+Job-router Lambda function will take an event from
+an AWS EventBridge rule and make a call to an endpoint
+for running scheduled jobs. These endpoints could be through
+a load balancer or directly to an ECS service
+"""
+import os
+import boto3
+import urllib3
+import jmespath
+
+client = boto3.client('ecs')
+ssm_client = boto3.client('ssm')
+
+service_port_map = {
+    'metadata-db' : '3001',
+    'ingest' : '3002',
+    'search' : '3003',
+    'indexer' : '3004',
+    'bootstrap' : '3006',
+    'virtual-product' : '3009',
+    'access-control' : '3011'
+}
```

Comment:
These ports should not be hard coded. We should read these from the parameter store, or environment variable, or something else. I know we have defaults hard coded in the CMR, which is fine, but we do first check to see if an environment variable or parameter store overrides the default.

Reply:
I had that there strictly for local dev. I'd put that in real early, but should pull it out since there's no reason to use the lambda in a local dev scenario.